ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -352,6 +352,16 @@ interface {{ port_channel_interface.name }}
 {%     elif port_channel_interface.bfd.echo is arista.avd.defined(false) %}
    no bfd echo
 {%     endif %}
+{%     if port_channel_interface.bfd.neighbor is arista.avd.defined %}
+   bfd neighbor {{ port_channel_interface.bfd.neighbor }}
+{%     endif %}
```

Comment:
If you want this to work for L2 Port-channels you need to introduce the global command `bfd local-address A.B.C.D` as well.

Reply:
Added local-address to router-bfd